learn/advanced/updating.md

```diff
@@ -1,30 +1,33 @@
 # Update to the latest Meilisearch version
 
-Currently, Meilisearch databases can only be opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to **migrate an existing database** from an older version of Meilisearch to the most recent one.
+:::warning
```

Comment:
I don't like how this looks:

<img width="780" alt="Screen Shot 2022-08-08 at 16 40 05" src="https://user-images.githubusercontent.com/90181761/183420144-922ab6ff-7e3e-49f1-9b07-8b71fbd425a4.png">

But I'm not sure what would be a better solution

Reply:
I think this should be split in two warnings. 

The first (guide  does not work on <v0.15) is in the right place. Perhaps we could move it to the end of the intro—might be more aesthetically pleasing not to have the warning right next to the page title. You could also use `::: danger` instead of `::: warning` if you want to be extra scary.

The second warning (auth header) could perhaps be leave where it was? Or perhaps just as the last line on the intro, as regular text? If you think that the problem is the code sample taking too much space, I think we could do without it.